.github/workflows/CI.yml

```
@@ -178,6 +165,32 @@ jobs:
                    else
                        packages+=(libgmp-dev libreadline-dev zlib1g-dev)
                    fi
+                   if [[ $TEST_SUITES == *testpackages* ]] ; then
+                       packages+=(
+                         4ti2
+                         libboost-dev
+                         libcdd-dev
+                         libcurl4-openssl-dev
+                         libfplll-dev
+                         libmpc-dev
+                         libmpfi-dev
+                         libmpfr-dev
+                         libncurses5-dev
+                         libzmq3-dev
+                         pari-gp
+                         singular
```

Since you asked for it:
```suggestion
                         4ti2                   # 4ti2Interface
                         libboost-dev           # NormalizInterface
                         libcdd-dev             # CddInterface
                         libcurl4-openssl-dev   # curlInterface
                         libfplll-dev           # float
                         libmpc-dev             # float
                         libmpfi-dev            # float
                         libmpfr-dev            # float
                         libncurses5-dev        # browse
                         libzmq3-dev            # ZeroMQInterface
                         pari-gp                # alnuth
                         singular               # singular
```